components/BottomNavigation/src/MDCBottomNavigationBarController.h

```diff
@@ -0,0 +1,68 @@
+#import <UIKit/UIKit.h>
+
+NS_ASSUME_NONNULL_BEGIN
```

Comment:
Please delete and update nullability accordingly - we use explicit nullability annotations throughout MDC.

Reply:
Done.